tests/specs/ecosystem/ayonli/jsext/0_7_10.test

```diff
@@ -13,8 +13,53 @@ ayonli/jsext/0.7.10
 -- stdout --
 
 -- stderr --
-error: TS2503 [ERROR]: Cannot find namespace 'NodeJS'.
+error: TS4114 [ERROR]: This member must have an 'override' modifier because it overrides a member in the base class 'Map<K, V>'.
+  get [Symbol.toStringTag](): "BiMap" {
+      ~~~~~~~~~~~~~~~~~~~~
+    at file://<tmpdir>/collections/BiMap.ts:4:7
+
+TS4114 [ERROR]: This member must have an 'override' modifier because it overrides a member in the base class 'Map<K, any>'.
+  get [Symbol.toStringTag](): "CiMap" {
+      ~~~~~~~~~~~~~~~~~~~~
+    at file://<tmpdir>/collections/CiMap.ts:2:7
+
+TS2416 [ERROR]: Property 'entries' in type 'CiMap<K, V>' is not assignable to the same property in base type 'Map<K, any>'.
+  Type '() => IterableIterator<[K, V]>' is not assignable to type '() => MapIterator<[K, any]>'.
+    Type 'IterableIterator<[K, V]>' is missing the following properties from type 'MapIterator<[K, any]>': map, filter, take, drop, and 9 more.
+  override entries(): IterableIterator<[K, V]> {
+           ~~~~~~~
+    at file://<tmpdir>/collections/CiMap.ts:20:12
```

Comment:
I guess these should be gone once  no `noImplicitOverride` for remote modules is in?